packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "vscode": "1.1.4"
   },
+  "extensionDependencies": ["salesforce.salesforcedx-vscode-apex"],
```

Comment:
This is only an install time dependency right? It's not enforced at runtime?

So, if you had installed both. But, if you then remove salesforcedx-vscode-apex, things will still load? Or does the debugger module stop loading then?